code/modules/projectiles/guns/projectile_gun.dm

```diff
@@ -49,13 +49,13 @@
 		. += knife_overlay
 
 /obj/item/gun/projectile/process_chamber(eject_casing = 1, empty_chamber = 1)
-	var/obj/item/ammo_casing/AC = chambered //Find chambered round
-	if(isnull(AC) || !istype(AC))
+	var/obj/item/ammo_casing/ammo_chambered = chambered //Find chambered round
+	if(isnull(ammo_chambered) || !istype(ammo_chambered))
```

Comment:
an isnull check isn't needed here as it will automatically fail the istype anyway if it is null

Reply:
:gatto: